keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1685,7 +1690,7 @@ Yes. Applications that were already deployed and are running will continue to
 work, but they will stop working when containers get restarted because those
 restarted containers won't have the additional resources.
 
-###### What happens if we reenable the feature if it was previously rolled back?
+###### What happens if we re-enable the feature if it was previously rolled back?
```

Comment:
This came straight from https://github.com/kubernetes/enhancements/tree/master/keps/NNNN-kep-template. Let's keep the original spelling.

Reply:
done